### ISSUE TITLE
Cancelled installation treated as installed by scoop

### ISSUE THREAD

**Issue by slaughtering** [lukesampson/scoop#4058]

**Comment by headhunterbr** [comment]
This is so old that it has become normal.

Apparently the developers don't know how to check if the ERRORLEVEL variable is equal to 0

**Comment by slaughtering** [comment]
> This is so old that it has become normal.
> 
> Apparently the developers don't know how to check if the ERRORLEVEL variable is equal to 0

Yeah manually uninstall everytime, maybe not comming up with a simple solution by the devs so far.

**Comment by niheaven** [comment]
https://github.com/lukesampson/scoop/pull/3281
